src/ansys/mechanical/core/embedding/viz/embedding_plotter.py

```diff
@@ -81,11 +82,5 @@ def to_pyvista_plotter(app: "ansys.mechanical.core.embedding.App"):
         pv_transform = _transform_to_pyvista(scenegraph_node.Transform)
         polydata = pv.PolyData(np_coordinates, np_indices).transform(pv_transform)
         color = pv.Color(bgr_to_rgb_tuple(body.Color))
-        plotter.add_mesh(polydata, color=color, smooth_shading=True)
+        plotter.plot(polydata, color=color, smooth_shading=True)
```

Comment:
Why does ansys visualization use the method `plot` for adding plot data to the plotter object?

Reply:
It's just a convention we decided. `plot` for adding objects in the plotter and `show` to pop the window and show those objects.